templates/common/_base/files/wait-for-ipsec-connect.yaml

```diff
@@ -0,0 +1,44 @@
+mode: 0755
+path: "/usr/local/bin/ipsec-connect-wait.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    set -x
+
+    if [ ! -e "/etc/ipsec.d/openshift.conf" ]; then
```

Comment:
@pperiyasamy on fresh cluster install I assume this will be the case?

Reply:
@tssurya right. this will be the case for ipsec external and disabled mode as well.